docs/contracts.rst

```diff
@@ -1383,6 +1383,6 @@ It is also possible to extend elementary types in that way::
     }
 
 Note that all library calls are actual EVM function calls. This means that
-if you pass memory or value types, a copy will be performed, even of the
+if you pass ``memory`` or value types, a copy will be performed, even of the
```

Comment:
Here, the term "memory" is not used as the name of something but rather the concept and thus I would not code-format it, same goes for storage below.

Reply:
Reverted